### PR TITLE
New design of the homepage

### PR DESCRIPTION
**Pull request by seisman**

### CODE REVIEW

File: content/home/about.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
++++
+
+### 这是什么？
+
+seismo-learn，即地震“学”。
+
+### 为什么要建这个网站？
+
+- 源起与 seisman 博客
```
Comment (core-man) [inline content/home/about.md:57]:
```suggestion
- 源起于 seisman 博客
```

File: content/home/about.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+- 系统整理相关材料
```
Comment (core-man) [inline content/home/about.md:58]:
```suggestion
- 系统整理和维护与地震学相关的材料
```

File: content/home/about.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+- 提供入门培训材料
```
Comment (core-man) [inline content/home/about.md:59]:
```suggestion
- 提供和维护地震学入门培训材料
```

File: content/home/about.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+### 我们是谁？
+
+- 姚家园 @ Nanyang Technological Universit
```
Comment (core-man) [inline content/home/about.md:63]:
```suggestion
- 姚家园（a.k.a. core-man） @ Nanyang Technological University
```

File: content/home/about.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+- 田冬冬 (a.k.a. seisman) @ Michigan State University
+
+### 如何参与？
+
+- github
```
Comment (core-man) [inline content/home/about.md:68]:
```suggestion
- [GitHub](https://github.com/seismo-learn)
```

File: content/home/about.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+- bug
```
Comment (core-man) [inline content/home/about.md:69]:
```suggestion
- Issue
```

File: content/home/about.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+- pull request
```
Comment (core-man) [inline content/home/about.md:70]:
```suggestion
- Pull Request
```

File: content/home/about.md [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+- discussions
```
Comment (core-man) [inline content/home/about.md:71]:
```suggestion
- GitHub Discussions
```

File: content/home/docs.md [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+[<i class="fab fa-github fa-2x"></i>](https://github.com/seismo-learn/seismology101)
+[<i class="fas fa-bug fa-2x"></i>](https://github.com/seismo-learn/seismology101/issues)
+[<i class="fas fa-comment fa-2x"></i>](https://github.com/seismo-learn/seismology101/discussions)
+
+{{% /col %}}
+
+{{% col "col-12 col-sm-4" %}}
+<div class="featurette-icon"><a href="https://seismo-learn.org/software/" target="_blank"><i class="fas fa-laptop-code fa-lg"></i></a></div>
+
+#### 地震学软件
```
Comment (core-man) [inline content/home/docs.md:70]:
```suggestion
#### 地震“学“软件
```

File: content/home/docs.md [hunk 10]
```diff
@@ -0,0 +1,93 @@
...
+[<i class="fas fa-bug fa-2x"></i>](https://github.com/seismo-learn/software/issues)
+[<i class="fas fa-comment fa-2x"></i>](https://github.com/seismo-learn/software/discussions)
+{{% /col %}}
+
+{{% col "col-12 col-sm-4" %}}
+<div class="featurette-icon"><a href="https://seismo-learn.org/seismology101/" target="_blank"><i class="fas fa-book-reader fa-lg"></i></a></div>
+
+#### 地震学教程
+
+📚 开源、不限篇幅、随时更新、人人可编辑的地震学参考书！
```
Comment (core-man) [inline content/home/docs.md:85]:
```suggestion
📚 人人可编辑的地震学参考书！开源、随时更新、不限篇幅！
```

File: content/home/hero.md [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+# Hero widget.
+widget = "hero"  # See https://sourcethemes.com/academic/docs/page-builder/
+headless = true  # This file represents a page section.
+active = true  # Activate this widget? true/false
+weight = 10  # Order that this section will appear.
+
+title = "地震“学”"
+
+# Hero image (optional). Enter filename of an image in the `static/media/` folder.
+hero_media = "hero-academic.png"
```
Comment (core-man) [inline content/home/hero.md:11]: Need to put a png in `/static`?

File: content/home/hero.md [hunk 12]
```diff
@@ -0,0 +1,58 @@
...
+# Note. An optional note to show underneath the links.
+[cta_note]
+  label = '<a class="js-github-release" href="https://sourcethemes.com/academic/updates" data-repo="gcushen/hugo-academic">Latest release<!-- V --></a>'
++++
+
+**The Best Way to Create the Website You Want from Markdown (or Jupyter/RStudio)**
+
+Build **Anything** with Widgets
+
+<span style="text-shadow: none;"><a class="github-button" href="https://github.com/gcushen/hugo-academic" data-icon="octicon-star" data-size="large" data-show-count="true" aria-label="Star this on GitHub">Star</a><script async defer src="https://buttons.github.io/buttons.js"></script></span>
```
Comment (core-man) [inline content/home/hero.md:58]:
```suggestion
# **The Best Way to Create the Website You Want from Markdown (or Jupyter/RStudio)**

# Build **Anything** with Widgets

# <span style="text-shadow: none;"><a class="github-button" href="https://github.com/gcushen/hugo-academic" data-icon="octicon-star" data-size="large" data-show-count="true" aria-label="Star this on GitHub">Star</a><script async defer src="https://buttons.github.io/buttons.js"></script></span>
```

**Comment by seisman** [comment]
It's just a draft version. We need to change the texts if we all agree that the current design is OK.

File: content/home/hero.md [hunk 13]
```diff
@@ -0,0 +1,58 @@
...
+  icon_pack = "fas"
+  icon = "download"
+
+[cta_alt]
+  url = "https://sourcethemes.com/academic/"
+  label = "View Documentation"
+
+# Note. An optional note to show underneath the links.
+[cta_note]
+  label = '<a class="js-github-release" href="https://sourcethemes.com/academic/updates" data-repo="gcushen/hugo-academic">Latest release<!-- V --></a>'
```
Comment (core-man) [inline content/home/hero.md:51]:
```suggestion
# [cta_note]
#  label = '<a class="js-github-release" href="https://sourcethemes.com/academic/updates" data-repo="gcushen/hugo-academic">Latest release<!-- V --></a>'
```

File: content/home/hero.md [hunk 14]
```diff
@@ -0,0 +1,58 @@
...
+
+# Call to action links (optional).
+#   Display link(s) by specifying a URL and label below. Icon is optional for `[cta]`.
+#   Remove a link/note by deleting a cta/note block.
+[cta]
+  url = "https://sourcethemes.com/academic/docs/install/"
+  label = "Get Started"
+  icon_pack = "fas"
+  icon = "download"
+
```
Comment (core-man) [inline content/home/hero.md:44]:
```suggestion
# [cta]
#  url = "https://sourcethemes.com/academic/docs/install/"
#  label = "Get Started"
#  icon_pack = "fas"
#  icon = "download"

```

File: content/home/hero.md [hunk 15]
```diff
@@ -0,0 +1,58 @@
...
+[cta_alt]
+  url = "https://sourcethemes.com/academic/"
+  label = "View Documentation"
```
Comment (core-man) [inline content/home/hero.md:47]:
```suggestion
# [cta_alt]
#  url = "https://sourcethemes.com/academic/"
#  label = "View Documentation"
```

**Comment by core-man** [comment]
> It's just a draft version. We need to change the texts if we all agree that the current design is OK.

I feel it's good and simple.

**Comment by seisman** [comment]
It's still unclear to me what to put here:

image

I'm thinking about some waveforms, beachballs, faults et al..

And also need to change the texts:

image

**Comment by core-man** [comment]
> It's still unclear to me what to put here:
> 
> image

> I'm thinking about some waveforms, beachballs, faults et al..

We can leave only “地震学” and logo here now.

> And also need to change the texts:
> 
> image

Just as simple as possible.

**Comment by seisman** [comment]
OK, will try to improve it a little.

**Comment by seisman** [comment]
Now it's ready for review.

File: README.md [hunk 16]
```diff
@@ -2,6 +2,25 @@
...
+
+- 网站引擎：[Hugo](https://gohugo.io/)
+- 网站主题：[Wowchemy](https://wowchemy.com/)
+
+## 构建网站
+
+1.	安装 [Hugo](https://gohugo.io/)
+2.	构建网站
+	```
+	hugo
```
Comment (core-man) [inline README.md:15]:
```suggestion
	$ hugo
```

File: README.md [hunk 17]
```diff
@@ -2,6 +2,25 @@
...
+	```
+	生成的网站位于 `public` 目录下。
+3.  也可以执行 `hugo server` 并在浏览器里打开 http://localhost:1313 以预览网站。
+
+## 更新主题
+
+执行如下命令以更新主题：
+
+```
+sh update_wowchemy.sh
```
Comment (core-man) [inline README.md:25]:
```suggestion
$ sh update_wowchemy.sh
```

File: content/home/about.md [hunk 18]
```diff
@@ -0,0 +1,61 @@
...
+[advanced]
+ # Custom CSS.
+ css_style = ""
+
+ # CSS class.
+ css_class = ""
+
++++
+
+### 我们是谁？
```
Comment (core-man) [inline content/home/about.md:51]:
```suggestion
### 维护者是谁？
```

**Review by core-man — APPROVED** [review]

**Review by core-man — APPROVED** [review]

**Pull request merged**